IntelliJ/Nx/build.gradle.kts

```diff
@@ -0,0 +1,176 @@
+plugins {
+    id("java")
+}
+
+group = "com.nuix.nx"
+version = "1.19.0-SNAPSHOT"
```

Comment:
Can you make most of these variables properties, with defaults please?  This makes them more compatible with Jenkins, but also lets you override on the command line (or IntelliJ run configuration.  An example for version would be:

`findProperty("version") ?: "1.19.0-SNAPSHOT"`

And to override the defaults you use:

`./gradlew build -Pversion=2.0`

Variables I would like to see as properties:
* version
* nuixEngineDirectory
* engineLibDir
* nuixAppLibDir
* nuixTempDir